src/test/ui/error-codes/E0606.stderr

```diff
@@ -2,13 +2,10 @@ error[E0606]: casting `&u8` as `u8` is invalid
   --> $DIR/E0606.rs:2:5
    |
 LL |     &0u8 as u8; //~ ERROR E0606
-   |     ^^^^^^^^^^ cannot cast `&u8` as `u8`
-   |
-help: did you mean `*&0u8`?
-  --> $DIR/E0606.rs:2:5
-   |
-LL |     &0u8 as u8; //~ ERROR E0606
-   |     ^^^^
+   |     ----^^^^^^
+   |     |
+   |     cannot cast `&u8` as `u8`
+   |     help: dereference the expression: `*&0u8`
```

Comment:
This is obviously not ideal, but at least clippy will lint against it?

Reply:
yea, seems fine to me.